src/applications/static-pages/view-modify-dependent/686-cta/create686CcontentReveal.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default function create686ContentReveal(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "HomepageBanner" */
```

Comment:
Ah, missed this - I don't think it hurts anything but you may want to update your `webpackChunkName` here in this comment

Reply:
oooh, good catch! Updated.